_integration/testsuite/install-contour-working.sh

```diff
@@ -39,30 +41,44 @@ kind::cluster::exists() {
     ${KIND} get clusters | grep -q "$1"
 }
 
-kind::cluster::load() {
+kind::cluster::load::docker() {
     ${KIND} load docker-image \
         --name "${CLUSTERNAME}" \
         "$@"
 }
 
+kind::cluster::load::archive() {
+    ${KIND} load image-archive \
+        --name "${CLUSTERNAME}" \
+        "$@"
+}
+
 if ! kind::cluster::exists "$CLUSTERNAME" ; then
     echo "cluster $CLUSTERNAME does not exist"
     exit 2
 fi
 
-# Build the current version of Contour.
-make -C ${REPO} container IMAGE=docker.io/projectcontour/contour VERSION="v$$"
-
-for t in $TAGS ; do
-    docker tag \
-        docker.io/projectcontour/contour:"v$$" \
-        docker.io/projectcontour/contour:$t
-done
-
-# Push the Contour build image into the cluster.
-for t in $TAGS ; do
-    kind::cluster::load docker.io/projectcontour/contour:$t
-done
+if [ "${LOAD_PREBUILT_IMAGE}" = "true" ]; then
+    IMAGE_FILE=$(ls ${REPO}/image/contour-*.tar)
+    VERSION=$(echo ${IMAGE_FILE} | sed -E 's/.*-(.*).tar/\1/')
+    kind::cluster::load::archive ${IMAGE_FILE}
+else
+    # Build the current version of Contour.
+    VERSION="v$$"
+    make -C ${REPO} container IMAGE=docker.io/projectcontour/contour VERSION=$VERSION
+
+    for t in $TAGS ; do
+        docker tag \
+            docker.io/projectcontour/contour:"v$$" \
+            docker.io/projectcontour/contour:$t
+    done
```

Comment:
this seems unnecessary now since we're updating the YAML to point to the specific version tag. Shouldn't hurt anything so not a big deal either way.